tests/test_document.py

```diff
@@ -38,33 +36,22 @@ def test_not_equals():
     assert doc1 != doc2
 
 
-def test_less_than():
-    doc1 = create_document(did='11111111-1111-1111-1111-111111111111')
-    doc2 = create_document(did='11111111-1111-1111-1111-111111111112')
-    assert doc1 < doc2
-
-    # reverse order docs
-    docs = [
-        create_document(did='11111111-1111-1111-1111-11111111111' + str(suffix))
-        for suffix in range(10, 0, -1)
-    ]
-    # sorted() sorts it in order from lowest to highest did because of __lt__/__gt__
-    assert sorted(docs) == docs[::-1]
-
-
-def test_greater_than():
+def test_greater_than_less_than():
     doc1 = create_document(did='11111111-1111-1111-1111-111111111111')
     doc2 = create_document(did='11111111-1111-1111-1111-111111111112')
     assert doc2 > doc1
+    assert doc1 < doc2
 
     # reverse order docs
     docs = [
         create_document(did='11111111-1111-1111-1111-11111111111' + str(suffix))
         for suffix in range(10, 0, -1)
     ]
     # sorted() sorts it in order from lowest to highest did because of __lt__/__gt__
-    assert sorted(docs) == docs[::-1]
-
+    did = sorted(docs)[0].did
```

Comment:
I don't think you should have to do this. It's probably safe to just remove the 10 from the `create_document` step. Adding another digit creates something other than a UUID and I don't know if we're going to use dids other than uuid.

One option is to strengthen the `__lt__` and `__gt__` magic methods to typecast the strings to UUIDs and use the UUID class' magic methods for comparisons. Doing this will also force the removal of the `10` suffix because it would not be a UUID and subsequently raise an exception trying to typecast to a UUID.